docker-e2e-compose.yml

```diff
@@ -0,0 +1,54 @@
+version: '3'
+
+services:
+  db:
+    image: postgres:11
+    environment:
+      - POSTGRES_DB=ab2d
+      - POSTGRES_USER=ab2d
+      - POSTGRES_PASSWORD=ab2d
+    ports:
+      - "5432:5432"
+  build:
+    image: maven:3-jdk-12
```

Comment:
shouldn't this be JDK 13?

Reply:
Yeah it should be 13, I copied it before I merged the latest master which now has 13.